bot/exts/evergreen/cheatsheet.py

```diff
@@ -0,0 +1,108 @@
+import random
+import re
+import typing as t
+from urllib.parse import quote_plus
+
+from discord import Embed
+from discord.ext import commands
+from discord.ext.commands import BucketType, Context
+
+from bot import constants
+from bot.constants import Categories, Channels, Colours, ERROR_REPLIES, Roles, WHITELISTED_CHANNELS
+from bot.utils.decorators import with_role
+
+ERROR_MESSAGE = f"""
+Unknown cheat sheet. Please try to reformulate your query.
+
+**Examples**:
+```md
+{constants.Client.prefix}cht read json
+{constants.Client.prefix}cht hello world
+{constants.Client.prefix}cht lambda
+```
+If the problem persists send a message in <#{Channels.dev_contrib}>
+"""
+
+URL = 'https://cheat.sh/python/{search}'
+ESCAPE_TT = str.maketrans({"`": "\\`"})
+ANSI_RE = re.compile(r"\x1b\[.*?m")
+
+
+class CheatSheet(commands.Cog):
+    """Commands that sends a result of a cht.sh search in code blocks."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @staticmethod
+    def fmt_error_embed() -> Embed:
+        """
+        Format the Error Embed.
+
+        If the cht.sh search returned 404, overwrite it to send a custom error embed.
+        link -> https://github.com/chubin/cheat.sh/issues/198
+        """
+        embed = Embed(
+            title=random.choice(ERROR_REPLIES),
+            description=ERROR_MESSAGE,
+            colour=Colours.soft_red
+        )
+        return embed
+
+    def result_fmt(self, url: str, body_text: str) -> t.Tuple[bool, t.Union[str, Embed]]:
+        """Format Result."""
+        if body_text.startswith("#  404 NOT FOUND"):
+            embed = self.fmt_error_embed()
+            return True, embed
+
+        body_space = min(1986 - len(url), 1000)
+
+        if len(body_text) > body_space:
+            description = f"**Result Of cht.sh**\n" \
+                          f"```python\n{body_text[:body_space]}\n" \
+                          f"... (truncated - too many lines)```\n" \
+                          f"Full results: {url} "
```

Comment:
```suggestion
description = ("**Result Of cht.sh**\n"
               f"```python\n{body_text[:body_space]}\n"
               "... (truncated - too many lines)```\n"
               f"Full results: {url} "
            )
```
This would look neater than the `\` escaping of the string I think. Also, if you do change this, please change it for the next `description` variable also. (Also, the formatting isn't very good in this so please fix it when you change it)

Reply:
Alright will do that!